cluster/gke.go

```diff
@@ -10,13 +10,13 @@ import (
 	"github.com/banzaicloud/pipeline/model"
 	"github.com/banzaicloud/pipeline/secret"
 	"github.com/banzaicloud/pipeline/utils"
-	"github.com/gin-gonic/gin"
 	"github.com/gin-gonic/gin/json"
 	"github.com/go-errors/errors"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
 	"golang.org/x/oauth2/google"
 	gke "google.golang.org/api/container/v1"
+	"google.golang.org/api/compute/v1"
```

Comment:
this one is wrong here, you can check out that v1 package is already imported in https://github.com/banzaicloud/pipeline/blob/53b3b1a82368291c2eef994c64145f1d32e6e2e7/cluster/gke.go#L23